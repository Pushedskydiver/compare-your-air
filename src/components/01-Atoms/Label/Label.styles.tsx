import styled from 'styled-components';

const Label = styled.label`
  position: absolute;
  width: 1px;
  height: 1px;
  clip: rect(1px, 1px, 1px, 1px);
  padding: 0;
  border: 0;
  overflow: hidden;
`;

export default Label;
