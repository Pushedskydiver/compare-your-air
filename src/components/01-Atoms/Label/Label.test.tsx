import React from 'react';
import { cleanup, render } from '@testing-library/react';
import Label from './Label.styles';

afterEach(() => {
  cleanup();
});

describe('<Label />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Label>This is a label</Label>
    );

    expect(container).toMatchSnapshot();
  });
})