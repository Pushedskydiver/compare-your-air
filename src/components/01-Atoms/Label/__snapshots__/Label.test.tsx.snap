// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> should match snapshot 1`] = `
<div>
  <label
    class="sc-AxjAm gDVftL"
  >
    This is a label
  </label>
</div>
`;
